File: unity/Assets/Hallway/Constructor/RPG/barrelJump.js
#pragma strict

var Bootstrap : GameObject;

function OnTriggerEnter() {
	var scr1 : RPGParser = Bootstrap.GetComponent.<RPGParser>();
	if (!scr1.RPG.barrelRoll) {
		scr1.Achievement("Сделана бочка!\n+10 очков!", 10);
		scr1.RPG.barrelRoll = true;
	}	
}

File: unity/Assets/Hallway/Constructor/RPG/barrelJump.cs
using UnityEngine;

public class barrelJump : MonoBehaviour
{

    public GameObject Bootstrap;
    private void OnTriggerEnter()
    {
        RPGParser rpgParser = Bootstrap.GetComponent<RPGParser>();
        if (!rpgParser.RPG.barrelRoll)
        {
//            rpgParser.Achievement("Сделана бочка!\n+10 очков!", 10);
//            rpgParser.RPG.barrelRoll = true;
        }
    }
}
